src/api/app/controllers/webui2/package_controller.rb

```diff
@@ -25,4 +28,11 @@ def webui2_statistics
                                                       repository: @repository,
                                                       architecture: params[:arch]).results
   end
+
+  private
+
+  def backend_linked_project_package(package)
+    lt = package.backend_package.links_to
```

Comment:
maybe we could come up with a better name than `lt` here? `link_to_package` ?

Reply:
Sure, I'm returning the project and package names, so what about `project_and_package_names`?

Reply:
great!

Reply:
Fixed